packages/cfd/src/Containers/__tests__/cfd-dashboard.spec.tsx

```diff
@@ -0,0 +1,548 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
avoiding a warning _'@testing-library/react' should be listed in the project's dependencies. Run 'npm i -S @testing-library/react' to add it_